.gitignore

```diff
@@ -2,4 +2,5 @@ phpunit.xml
 composer.lock
 build
 vendor
-coverage.clover
\ No newline at end of file
+coverage.clover
+ 
```

Comment:
Looks like you are ignoring files that only have a single space character (`U+0020`) in the name.